simulations/vip-181/simulations.ts

```diff
@@ -0,0 +1,81 @@
+import { TransactionResponse } from "@ethersproject/providers";
+import { expect } from "chai";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { expectEvents } from "../../src/utils";
+import { forking, testVip } from "../../src/vip-framework";
+import {
+  CORE_POOL_COMPTROLLER,
+  LIQUID_STAKED_BNB_COMPTROLLER,
+  MATIC_NEW_BORROW_CAP,
+  MATIC_NEW_SUPPLY_CAP,
+  SNBNB_NEW_BORROW_CAP,
+  SNBNB_NEW_SUPPLY_CAP,
```

Comment:
I'd rather not reuse values from the code in tests, it kinda undermines the purpose of tests IMO

Reply:
I agree, let's try to avoid it in the future.